README.md

```diff
@@ -100,20 +97,20 @@ At second shell/prompt, watch files for changes to rebuild:
 ### In Firefox:
 
 1. `about:debugging > [load temporary addon] > choose `dist/addon.xpi`
-2. `tools > Web Developer > Browser Toolbox`
+2. `tools > Web Developer > Browser Toolbox`.
 
 
 ### Description of architecture
 
-Restartless (`bootstrap.js`) extension.
+This addon is structured as a restartless (`bootstrap.js`) extension.
```

Comment:
Let's expand this a little bit highlighting what the main files are for. Let's do this as a bullet list:

- `HomepageStudy.jsm`: implements the logic to extract the eTLD+1 from the preference and apply RAPPOR.
- `TelemetryRappor.jsm`: implements the RAPPOR algorithm and the related utility functions.
- `StudyUtils.jsm`: miscellaneous SHIELD utils.
- `bootstrap.js`:  contains addon specific boilerplate code.